### ISSUE TITLE
A question about data format

### ISSUE THREAD

**Issue by seongminp** [gcunhase/AMICorpusXML#6]
Hello. First of all thank you for this wonderful library.

I have a question about the transcription format. 

For each `.txt` file in [`data/ami-transcripts`](https://github.com/gcunhase/AMICorpusXML/blob/master/data/ami-transcripts/EN2001a.transcript.txt), does each line denote a single speaker?

For example, would each line of [EN2001a](https://github.com/gcunhase/AMICorpusXML/blob/master/data/ami-transcripts/EN2001a.transcript.txt) belong to a separate speaker, of which there are 5?

Thank you!

**Comment by saprativa** [comment]
I don't think so. There are very few newline characters (4-5) in the .txt transcript files although the meetings had a very large number of speaker turns.

**Comment by saprativa** [comment]
Yes you are correct. But this doesn’t preserve the speaker turns from the actual meetings. And I also found some discrepancies in the transcript files and the speaker transcript files for ES2002c. But I need to check and verify again.
